relay-general/src/protocol/contexts/replay.rs

```diff
@@ -0,0 +1,55 @@
+use crate::protocol::EventId;
+use crate::types::{Annotated, Object, Value};
+
+/// Replay context
```

Comment:
Some more information would be nice here, see for example 

https://github.com/getsentry/relay/blob/8afa67264a69d5d9105264bf94254aa0bfb10dd8/relay-general/src/protocol/contexts/os.rs#L6-L9